src/utils.js

```diff
@@ -16,13 +19,23 @@ export function randomString(
   return result;
 }
 
+/**
+ * [safeSlot description]
+ *
+ * @param  {[type]} h    hyperscript markup from Vue render function
+ * @param  {[type]} slot component slot
+ * @return {[type]}      Slot or div wrapping the slot
+ */
 export function safeSlot(h, slot) {
   return slot && slot.length > 1 ? h('div', slot) : slot;
 }
 
 /**
- * @param {HTMLElement} el
- * @param {Event} event
+ * [applyFocusTrap description]
+ *
+ * @param  {HTMLElement} el    [description]
```

Comment:
Replace `[description]` with a useful description of the variable.

Reply:
So interesting. Did eslint automatically add these `[description]` blocks or was that you by hand?

Reply:
In Sublime Text > Install `DocBlockr` plugin > type `/**` then press tab on the line above a function. It will auto-generate the JSDoc block with placeholders. Type in the description, tab, type, tab, type, etc.

Reply:
But you can also run `npm run fix` and it will add in some basic stuff automatically. However if there was stuff already there that the linter didn't like, the auto-fix looks weird